apps/ledger-live-mobile/src/components/DeviceAction/index.tsx

```diff
@@ -78,13 +156,14 @@ export function DeviceActionDefaultRendering<R, H, P>({
   status,
   request,
   payload,
-}: Props<R, H, P>): JSX.Element | null {
+}: Props<H, P> & {
+  request?: R;
+}): JSX.Element | null {
   const { colors, dark } = useTheme();
   const dispatch = useDispatch();
-  const theme = dark ? "dark" : "light";
+  const theme: "dark" | "light" = dark ? "dark" : "light";
```

Comment:
I think we can have a `type AppTheme = "dark | light"` coming from`colors.ts`

Reply:
V2